zendesk_ticket_metrics/sla_metric_by_ticket

```diff
@@ -0,0 +1,75 @@
+CREATE VIEW zendesk.sla_metric_by_ticket AS (
+  with ticket_metric_ranked as (
+   select 
+    instance_id,
+    metric,
+    time,
+    sla_policy_id,
+    ticket_id
+   from (
+     select 
+      instance_id,
+      metric,
+      time,
+      sla_policy_id,
+      ticket_id,
+      row_number() over (partition by instance_id,metric,ticket_id order by time asc) rank
+     from zendesk.ticket_metric_event
+     where type='apply_sla'
+   ) as ticket_metric_event
```

Comment:
We are eliminating this table - why is the query based on this?  Recall I asked you to create a `ticket_sla_policy` VIEW on top of `ticket_metric_event`

Reply:
I was under the impression that `sla_policy_breach` was the one that was being removed. I'll put some time on your calendar so we can review.